src/components/Button/ButtonKitchenSink.tsx

```diff
@@ -241,6 +241,15 @@ function Buttons(): ReactElement {
 				<Button color="gray" variant="icon" mIcon={CloudDownloadIcon} onClick={clickHandler}></Button>
 				<Button color="gray" variant="icon" mIcon={MoreHorizIcon} onClick={openDropdown}></Button>
 			</div>
+
+			<h2>White Icon Buttons</h2>
+			<div style={{ background: 'black' }}>

```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/simpleviewinc/sv-mosaic/pullRequest?prid=9446470)
```suggestion
			<div style={{ background: "black" }}>
```